### DISCUSSION TITLE
【QA4】What kind of tool is SoloX?（SoloX是个什么样的工具？）

### DISCUSSION THREAD

**Discussion by rafa0128** [smart-test-ti/SoloX#110] [Q&A]
First of all, SoloX will only focus on performance testing, based on ease of use, free and lightweight, and will not build wheels repeatedly; Secondly, it basically imitates perfdog, but it must not replace perfdog. I am in favor of the commercialization of perflog. Such a large cost cannot always be public welfare, which is unfair. At the same time, I also hope that SoloX can become a booster for the continuous progress of perflog.

首先SoloX只会专注于性能测试，立足于好用、免费、轻量，不会重复造轮子；其次它的出现基本是模仿perfdog，但是一定不能替代perfdog。我是赞同perfdog走向商业化的，这么大的成本不可能永远是公益性的，这不公平。同时，我也希望SoloX能成为倒逼perfdog不断进步的助推剂。

**Comment by baoyuyong** [comment]
你好，请问我的测试报告没有数据是什么原因啊，期待回复

**Reply by rafa0128** [reply]
> 你好，请问我的测试报告没有数据是什么原因啊，期待回复

更新一下新版本，后续请到issue模块提问题

**Reply by xlg1987** [reply]
期待出一个视频介绍使用，安装。这样推动更加多人使用。期待你的回复，我的微信13421841858@rafa0128

**Reply by xlg1987** [reply]
Metersphere自动化测试工具介绍视频介绍了解一下。简单的使用安装这边没有看懂。
